OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/CreateKeyEmailFragment.java

```diff
@@ -137,6 +134,10 @@ public void onClick(View v) {
                     addEmail();
                 }
             });
+
+            if (mCreateKeyActivity.mAdditionalEmails != null) {
+                mEmailAdapter.addAll(mCreateKeyActivity.mAdditionalEmails);
+            }
```

Comment:
Please remove these changes from this pull request, they were already fixed in development branch.